src/LoginPage/slice.js

```diff
@@ -30,12 +48,68 @@ const { actions, reducer } = createSlice({
         },
       };
     },
+
+    setRefreshToken(state, { payload: refreshToken }) {
+      return {
+        ...state,
+        refreshToken,
+      };
+    },
+
+    setAccountInfo(state, { payload: uid }) {
+      const { accountInfo } = state;
+      return {
+        ...state,
+        accountInfo: {
+          ...accountInfo,
+          uid,
+        },
+      };
+    },
   },
 });
 
 export const {
+  setIsModalOpen,
   setIsLogin,
   changeLoginFields,
+  setRefreshToken,
+  setAccountInfo,
 } = actions;
 
+export function requestLogin() {
+  return async (dispatch, getState) => {
+    const {
+      login: {
+        loginFields: { email, password },
+      },
+    } = getState();
+
+    try {
+      const data = await postLogin({ email, password });
```

Comment:
```suggestion
const { refreshToken, localId: uid } = await postLogin({ email, password });
```
저는 요렇게 한방에 적어주는게 조금더 직관적이기는 하더라구용? 참고하세용!

Reply:
엇 그렇네요! data를 변수 선언해서 받는 과정을 넘어갈 수 있겠군요 ㅎㅎ
감사합니당